### ISSUE TITLE
nearly all person detection results

### ISSUE THREAD

**Issue by DAYHU** [experiencor/basic-yolo-keras#4]
hi, thanks for the awesome job and I have a problem when I do the process as the ipynb, the loss will nearly stop at around 5.7 and the detection on the images are nearly all persons when I lower the threshold to see more detection boxes and I cannot get the result like you show. I don't know why and do you have some ideas? Do I need train longer time? but it seems that output will be all persons.
Thanks so much~

![dog1](https://user-images.githubusercontent.com/28506423/29236489-fac578d6-7ed8-11e7-8355-6877d67b281a.jpg)
![dog2](https://user-images.githubusercontent.com/28506423/29236492-008d2caa-7ed9-11e7-87ce-876c8f13a7f5.jpg)

**Comment by experiencor** [comment]
Are you using the same loss scales as mine (SCALE_NOOB, SCALE_CONF, SCALE_COOR, SCALE_PROB = 0.5, 5.0, 5.0, 1.0)? If that is the case, loss of 5.7 is too high. I think you should reduce the base learning rate lr of SGD, says 1e-6 to further improve the loss.

**Comment by DAYHU** [comment]
@experiencor yeah, I don't change anything except path. you mean if I learning a longer time with a lower learning rate, I can get the result as you gave?
thanks~

**Comment by experiencor** [comment]
@DAYHU yes, that's what I mean! You should get the loss somewhere near 3.8 for the detection results to make sense.

**Comment by DAYHU** [comment]
and could you please tell me how many epoch it takes you to train to make the loss near 3.8 with 1e-5 or 1e-6? I am not sure which lr you use.Thanks so much !

**Comment by experiencor** [comment]
If my memory is correct, I initially used 1e-3 or 1e-4. Then I reduced to 1e-5 to obtain the loss near 3.8. You may try the same method.

**Comment by DAYHU** [comment]
ok, I will try. Thanks so much!

**Comment by DAYHU** [comment]
@experiencor I train like you do use a high lr in the beginning and it works! Thanks so much!
